dashboard/src/components/rmrk/Gallery/Gallery.vue

```diff
@@ -36,12 +36,13 @@
               <div class="card-content">
                 <p
                   v-if="!isLoading"
-                  class="title is-4 has-text-centered">
+                  class="title is-4 has-text-centered"
+                  :title="nft.name">
                   <router-link v-if="nft.count < 2" :to="{ name: 'nftDetail', params: { id: nft.id }}">
-                    {{ nft.name }}
+                    {{ truncateNFTName(nft.name) }}
```

Comment:
Please make vue filter out it e.g
`nft.name | truncate`

Reply:
Should be okay now